packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -22,7 +22,7 @@ import MediaUtil from '../media/util';
 import WebRTCStats from '../stats/index';
 import StatsMetrics from '../stats/metrics';
 import StatsUtil from '../stats/util';
-import ReconnectionError from '../common/errors/reconnection';
+// import ReconnectionError from '../common/errors/reconnection';
```

Comment:
Commented out?

Reply:
This is no longer used and creates a linting error [unused vars], I can remove it if needed.